src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -737,7 +797,8 @@ For example:
 {
    "PhaseEncodingDirection": "j-",
    "TotalReadoutTime": 0.095,
-   "IntendedFor": "func/sub-01_task-motor_bold.nii.gz"
+   "IntendedFor": "func/sub-01_task-motor_bold.nii.gz",
+   "B0FieldIdentifier": "pepolar_fmap0"
```

Comment:
In link with comment by @tsalo, it seems it used as prefix in this PR the fieldmap type as defined in section Fieldmap data (current line 607), so should not we use of `epi_fmap0` instead of `pepolar_fmap0`?

Reply:
Sure, the idea for these identifiers is that they must be unique within-subject. There's really no difference between `epi_fmap0` and `pepolar_fmap0`. However `epi_fmap0` could confuse some user that a valid raw suffix is necessary. I'm going to change that in line 692.